test/core/sup_tree_core/tmpdir_tracker_test.exs

```diff
@@ -0,0 +1,16 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule AntikytheraCore.TmpdirTrackerTest do
+  use ExUnit.Case
```

Comment:
In `sup_tree_core/*_test.exs`, `use Croma.TestCase` is used instead of `use ExUnit.Case`.
For a sense of unity, let's use `Croma.TestCase` here as well.

Reply:
Replaced with `Croma.TestCase`. 63d3b2f